### ISSUE TITLE
SDN node startup fail

### ISSUE THREAD

**Issue by benjaminapetersen** [openshift/origin#15311]
Seen in #15247 [https://ci.openshift.redhat.com/jenkins/job/test_pull_request_origin/3213/console](https://ci.openshift.redhat.com/jenkins/job/test_pull_request_origin/3213/console)

```
Jul 17 21:18:06 nettest-node-2 openshift[810]: F0717 21:18:06.343565     810 node.go:309] error: SDN node startup failed: failed to get subnet for this host: nettest-node-2, error: timed out waiting for the condition
```
@stevekuznetsov @enj

**Comment by enj** [comment]
@openshift/networking

**Comment by danwinship** [comment]
If this doesn't happen again, it was probably fixed by #15353.

**Comment by danwinship** [comment]
> If this doesn't happen again, it was probably fixed by #15353.

it didn't happen again
